src/libchessdetector/non_max_sup_pts.cpp

```diff
@@ -360,6 +360,8 @@ int non_max_sup_pts(size_t w, size_t h, int16_t image[], int border,
 	// prune the neighbourhood weaklings
 	int culled = cull_neighbourhood(num_found, cp, cn_halfwidth);
 
+	free(*pt_output);
+
 	// allocate storage for the accepted points
 	if (!(*pt_output = new_pt_output(num_found - culled))) {
```

Comment:
A better fix would be that new_pt_output use realloc instead of malloc. The `free(*pt_output);` would be then unnecessary. Still, I will not block the pull request, but a new issues should be created,

Reply:
You're right!

The issue was created here : https://github.com/IRCAD-IHU/hybrid_marker_track/issues/6